src/main/java/edu/depaul/cdm/se452/d2l_mock/student/Student.java

```diff
@@ -1,21 +1,29 @@
-package edu.depaul.cdm.se452.d2l_mock;
+package edu.depaul.cdm.se452.d2l_mock.student;
 
 import edu.depaul.cdm.se452.d2l_mock.discussion_thread.Poster;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Entity;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 @Data
+@Entity
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
 public class Student implements Poster {
```

Comment:
Lets not use Poster here and instead extend a User class. Using a similar example that the professor used in the last lecture